api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Suite.java

```diff
@@ -191,7 +208,7 @@ public String toString() {
         .toString();
   }
 
-  public void setVersion(long version) {
+  public void setVersion(Long version) {
```

Comment:
is it expected that `version` could be null?

Reply:
Yes,
I remember that it is strongly connected with updating version of a suite. Suite version is null during the first run of and typical run but during rerun, the suite has a version, so I needed to allow on setting empty version and set the same state like in a typical run. I didn't see a better way to set the correct suite's version during the rerun.